Bulldozer.CSV/CSVInstance.cs

```diff
@@ -28,108 +28,6 @@ public class CSVInstance
     {
         #region Enums
 
-        /// <summary>
```

Comment:
@guymaness2 this is quite the significant change, was this thoroughly tested? I think I understand that you are now just parsing directly to actual Rock Enums if you can, just hard coded strings are an interesting choice. Do we trim values and such to ensure that they get handled appropriately, even if they are valid? May have already been an issue casting to an enum, but I am curious if those are more forgiving on trimming values.